site_scons/daos_build.py

```diff
@@ -235,8 +227,16 @@ def configure_mpi(env):
         if not load_mpi(mpi):
             continue
         if _find_mpicc(env):
-            print("%s is installed" % mpi)
+            print(f'{mpi} is installed')
             return True
-        print("No %s installed and/or loaded" % mpi)
+        print('No {mpi} installed and/or loaded')
     print("No MPI installed")
     return False
+
+
+def setup(env):
+    """Add daos specific methods to environment"""
+    env.AddMethod(_add_build_rpath, 'd_add_build_rpath')
+    env.AddMethod(_configure_mpi, 'd_configure_mpi')
+    env.AddMethod(_run_command, 'd_run_command')
+    env.AddMethod(_add_rpaths, 'd_add_rpaths')
```

Comment:
This is the real change in this PR, the rest is just fluff.

1. Is this worthwhile?
2. do we want a common 'd_' prefix for all our methods to make it clear they're non-standard.
3. Should we copy the CamelCase naming of existing methods?
4. Assuming this is worthwhile should we do it for all daos_build functions (which will touch more code than this PR).

Reply:
For #2, I think we already have other precedent where we add stuff and don't name it specially.  For instance, CheckFlag and CheckFlagCC.   I think it would be fine to do something like AddBuildRPATH

Reply:
That would mean that we'd replace daos_build.library() with env.Library(), yet there is already a method with that name, the one that we're wrapping/

Reply:
You could do something like DAOSLibrary, I suppose